Controllers/ToppingsController.js

```diff
@@ -5,9 +5,43 @@ const ObjectId = require("mongodb").ObjectId;
 
 const getAllToppings = (req, res) => {
   DbService.connectToDb(async (db) => {
-    const toppings = await ToppingsService.getAllToppings(db);
-    res.json(toppings);
+    const result = await ToppingsService.getAllToppings(db);
+    let jsonRes = ResponsesService.successful();
+    jsonRes.data = result;
+    return res.json(jsonRes);
+  });
+};
+
+const updateTopping = (req, res) => {
+  const data = {
+    id: ObjectId(req.body.id),
+    wins: req.body.wins,
+    battles: req.body.battles,
+  };
```

Comment:
Strongly suggest running this object through some validation - this is my preferred tool for the job https://ajv.js.org/guide/getting-started.html

The example under Basic data validation is pretty much what you would need to do for this case.